src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -1517,7 +1517,6 @@ MulticopterPositionControl::task_main()
 
 						/* if we suddenly fall, reset landing logic and remove thrust limit */
 						if (hrt_elapsed_time(&_landing_started) > 15e5
-								&& _landing_thrust < FLT_EPSILON
 								/* XXX: magic value, assuming free fall above 4m/s2 acceleration */
 								&& (_acc_z_lp > 4.0f
```

Comment:
@AndreasAntener @LorenzMeier We should discuss these magic values here. Might be worth it to have them on the safe side.